Helper/SeedData.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Helpers.Http;
+using System.Net;
+
+namespace Helpers
+{
+    public static class SeedData
+    {
+        private const int TIMEOUT = 10000;
+        private const int RETRYLIMIT = 10;
+        private static HttpClient httpClient = new HttpClient();
+
+        public static void WaitForSeedComplete()
+        {
+            bool isSeedComplete = false;
```

Comment:
is the expectation here that if seed data doesn't complete within the allotted time/retries the tests will continue on? Shouldn't they fail?

Reply:
The tests will continue. But they will eventually fail, since there is no seeded data.

Reply:
why not preemptively fail with a nice message so it's clear what went wrong?